Packs/CommonScripts/Scripts/ScheduleGenericPolling/ScheduleGenericPolling.py

```diff
@@ -50,9 +50,19 @@ def calculate_end_time(timeout):
     return end_time.strftime(short_format)
 
 
-def main():
+def is_value_sanitized(value):
+    arg_names = ['ids', 'pollingCommand', 'pollingCommandArgName',
```

Comment:
We better not include the short string ids since it
Can appear in different strings

Reply:
Agree. Removed.

Reply:
https://github.com/demisto/content/pull/30123/commits/75dfcb764371d6e2aa8e2a4de53dfb0c7fbb1a3b